### PR TITLE
chore: update dotnet docs to add otel collector method

### PR DESCRIPTION
**Pull request by zriyansh**

### CODE REVIEW

File: docs/instrumentation/dotnet.md [hunk 1]
```diff
@@ -126,6 +135,104 @@ In the SigNoz account, open the `Services` tab. Hit the `Refresh` button on
...
+<figure data-zoomable align='center'>
+    <img src="/img/docs/sample_net_app.webp" alt="The sample .NET application is being monitored in the SigNoz ‘Services’ tab"/>
+    <figcaption><i>The sample .NET application is being monitored in the SigNoz ‘Services’ tab</i></figcaption>
+</figure>
+
+---
+
+### Send traces via OTel Collector binary
+
+**Step 1: Creating a new .NET Core Application**
```
Comment (ankit01-oss) [inline docs/instrumentation/dotnet.md]:
```suggestion
**Step 1: Creating a new .NET Core Application**
```

Let's remove this step. In docs we assume that the user will have their own app.

File: docs/instrumentation/dotnet.md [hunk 2]
```diff
@@ -126,6 +135,104 @@ In the SigNoz account, open the `Services` tab. Hit the `Refresh` button on
...
+
+
+**Step 4. Running the .NET application:**
+
+```bash
+dotnet build
+dotnet run
+```
+
+**Step 5: Setup OpenTelemetry binary as an agent in your machine**
```
Comment (ankit01-oss) [inline docs/instrumentation/dotnet.md]:
```suggestion
**Step 5: Setup OpenTelemetry collector binary as an agent in your machine**
```

Can we move this to step 1. The otel collector should be running first.

Comment (zriyansh) [reply docs/instrumentation/dotnet.md]: sure

File: docs/instrumentation/dotnet.md (hunk 1 above)
Comment (zriyansh) [reply docs/instrumentation/dotnet.md]: okay!

**Pull request merged**